### ISSUE TITLE
There is no laser reciever/detector

### ISSUE THREAD

**Issue by Adrodoc** [Adrodoc55/MapMakerHelper#3]
There should be a laser reciever/detector, like in the ApertureCraft Vanilla Map.

**Comment by HectorTheSquid** [comment]
Please tell me a way to detect the laser i want to do my laser chambers already! Can't you find a way of detecting the armor stands? I mean they're entities any testfor command or execute should find it
;c I'll try it myself....

**Comment by HectorTheSquid** [comment]
Okay whatever it does actually work with the testfor command just using this simple command /testfor @e[-59,12,669,1,type=ArmorStand] It found AVC_Laser so i guess you just not adding it because you are busy? :3 Anyways keep up the gr8 work

**Comment by Adrodoc** [comment]
@HectorTheSquid yes you are right, it is not difficult to add a laser detector, but I am currently not working on this project. The official adventure map for ACV contains laser detectors too. Using `/testfor @e[name=ACV_Laser]` is the correct way to go, you just gotta be careful with the frequency of ArmorStands. Using a 20hz clock with this testfor command will not give a constant redstone signal. Here is a screenshot of the pulse extender I used in my map.
I actually used the command `/execute @e[name=ACV_Laser,r=5] ~ ~ ~ /testforblock ~ ~ ~ minecraft:piston_extension` and placed a piston extension at the recieving block, but using the coordinates should work just as well.
![2016-06-22_21 30 44](https://cloud.githubusercontent.com/assets/15145501/16280599/b241c9f2-38c0-11e6-9c32-347ce3068bb3.png)

**Comment by HectorTheSquid** [comment]
Okay! Thanks for the info, i'll do my best on my map :D

2016-06-22 21:33 GMT+02:00 Adrodoc55 notifications@github.com:
